app/views/campaigns/_smtp_settings.html.erb

```diff
@@ -107,7 +107,6 @@
           <div class="col-xs-8">
             <%= ff.password_field(:smtp_password,
                   placeholder: 'password',
-                  value: @campaign.email_settings.smtp_password,
```

Comment:
Afaik this will mean whenever you save the template settings you will wipe the password unless you re-enter it each time. 

https://github.com/pentestgeek/phishing-frenzy/pull/185